guides/19_Video_Codecs.md

```diff
@@ -4,6 +4,12 @@ After raw video frames are captured (e.g. from a camera outputting an RGBA pixel
 
 The Amazon Chime SDK for Javascript supports setting a list of video codec preferences which will be used to select the encoder for an individual client. The API allows setting fallback codecs to, if desired, ensure that all participants in the call can receive video from senders as long as there is some intersection between the chosen codecs and the receive capabilities. This document will cover how to use the API, its effects, and some possible use cases for using it.
 
+## Video Codec List
+The supported video codecs in the Amazon Chime SDK client libraries include H264 CBP, VP8, VP9 and AV1.  AV1 is the best in terms of coding efficiency with the highest CPU consumption. AV1 has coding tools designed specifically for screen content, such as Palette mode, Intra Block Copy, and Identity Transform. For content sharing, AV1 gives about 60% bit reduction compared to H264. VP9 gives ~30% coding efficiency improvement than H264 CBP with better network adaption from multiple layers of bitstreams. 
```

Comment:
I am going to do a bit of refactoring of these guides in a PR before release and can include that information. But since that is not needed for initial test builds will hold off on that for now.